docs/pages/admin-guides/management/guides/scim/scim.mdx

```diff
@@ -0,0 +1,109 @@
+---
+title: SCIM – Access List Membership Delegation to SCIM providers
+description: How to manage Access List membership using  SCIM integration
+
+---
+
+The SCIM integration between **SCIM providers** and **Teleport** enables automated
+synchronization of SCIM group memberships and Teleport Access List memberships.
+This integration supports centralized identity governance in
+external Identity Management System (like SailPoint) while Teleport enforces
+fine-grained access controls defined by Access Lists membership grants types.
+
+User permissions in Teleport are defined through Access Lists. While role definitions
+live in Teleport, group membership is dynamically managed by SCIM Provider via SCIM group
+membership. This ensures users have up-to-date access aligned with organizational
+policies.
+
+## How it works
+
+The SCIM integration uses a 1:1 mapping between SCIM group and Teleport
+Access Lists
+
+- Each SCIM group `displayName` must match the `spec.title` of a Teleport
+Access List.
+- Access Lists of SCIM type must be created in advance in Teleport. In this guide we're creating them using Terraform.
+- Only Access Lists of type `scim` can be managed by SCIM providers.
+- Role assignments are handled in Teleport, while group membership is delegated to External Identity Management System (like SailPoint).
+
+## Prerequisites
+- Teleport Enterprise v(=teleport.version=)
```

Comment:
The version variable is going to be misleading once we publish v19. Let's hardcode the minimum version instead.